grype/db/v4/store/store.go

```diff
@@ -19,31 +19,22 @@ type store struct {
 	db *gorm.DB
 }
 
+func models() []any {
+	return []any{
+		model.IDModel{},
```

Comment:
should these be pointers? (the prior AutoMigrate calls appear to be passed pointers) (And same comment for other similar changes.)

Reply:
These are intended to be passed to automigrate, which only needs instances, not pointers. If being passed to `db.Create` where the instance represents a row in the DB, then a pointer is required (which is not intended with `Models()`.

`InitialData()` does return pointers since they represent rows to be added.